pkg/dashboard/static/revisions-view.js

```diff
@@ -32,7 +32,7 @@ function fillChartHistory(data, namespace, name) {
             $("#specRev").val(elm.revision).data("last-rev", elm.revision).data("last-chart-ver", elm.chart_ver)
         }
 
-        const rev = $(`<li class="px-2 pt-5 pb-4 mb-2 rounded border border-secondary bg-secondary position-relative">
+        const rev = $(`<li class="px-2 pt-5 pb-4 mb-2 rounded border border-secondary bg-secondary position-relative link">
```

Comment:
If I select text in rev row - will it cause reaction of click?

Reply:
nope, it just selects the text
Can you try it on your local and see if the change meets the expectation?

Reply:
Fixed in next patch